compiler/luci-interpreter/src/kernels/Concatenation.test.cpp

```diff
@@ -78,6 +78,42 @@ TEST(ConcatenationTest, Float)
   }
 }
 
+TEST(ConcatenationTest, Unsupported_Configure_Type_NEG)
+{
+  std::vector<int8_t> input1_data{1, 2, 3, 4, 5, 6};
+  std::vector<int8_t> input2_data{7, 8, 9, 10, 11, 12};
+  Tensor input1_tensor = makeInputTensor<DataType::S8>({2, 3}, input1_data);
+  Tensor input2_tensor = makeInputTensor<DataType::S8>({2, 3}, input2_data);
+  Tensor output_tensor = makeOutputTensor(DataType::S8);
+  ConcatenationParams params{};
+
+  // Try different 'axis' and expect different results.
```

Comment:
I can see only one axis value.

Reply:
Oh, sorry, I forget to remove this from copy test on above. I will remove this. sorry for confusing!